colors/build.js

```diff
@@ -2,29 +2,50 @@ const fs = require('fs');
 const colors = require('./photon.json');
 const metadata = require('./package.json');
 
-function createColor(color, element) {
-  var rv = [];
+function createColor(color, element, outputType) {
+  var output = [];
+  let prefix;
+  let lastColor;
+
+  switch(outputType) {
+    case 'scss':
+      prefix = '$';
+      break
+    case 'css':
+    default:
+      prefix = '  --';
+  }
+
   for (var variant in element) {
     if (element.hasOwnProperty(variant)) {
       var value = element[variant];
-      rv.push(`  --${color}-${variant}: ${value};\n`);
+      output.push(`${prefix}${color}-${variant}: ${value};\n`);
     }
   }
-  return rv;
+  return output;
 }
 
-let rv = [`/* Firefox Colors CSS Variables v${metadata.version} */
+
+let cssOutput = [`/* Firefox Colors CSS Variables v${metadata.version} */
 
 :root {
 `];
+
+let sassOutput = [`/* Firefox Colors SCSS Variables v${metadata.version} */
+
+`];
+
 for (var color in colors) {
   if (colors.hasOwnProperty(color)) {
     var element = colors[color];
-    rv.push(...createColor(color, element));
+    cssOutput.push(...createColor(color, element));
+    sassOutput.push(...createColor(color, element, 'scss'));
+    // Add a newline for scss when changing color.
+    sassOutput.push('\n');
   }
 }
-rv.push('}\n');
-fs.writeFile('colors.css', rv.join(''), 'utf8', (err) => {
-  if (err) throw err;
-  // console.log('It\'s saved!');
-});
+
+cssOutput.push('}\n');
+
+fs.writeFileSync('colors.css', cssOutput.join(''), 'utf8');
+fs.writeFileSync('colors.scss', sassOutput.join(''), 'utf8');
```

Comment:
Can do, I don't see much need to get hung up on using synchronous io vs async in a simple build tool like this though.